frontend/src/modules/placement/reducers.ts

```diff
@@ -43,6 +43,7 @@ export interface PlaceAllSourcePayload {
 const initialState: PlacementState = {
     placementType: PlacementType.GROUP,
     placementDirection: PlacementDirections.COLUMN,
+    gaps: [0, 0],
```

Comment:
You should use constants with names that explain their content for [0, 0] and [1, 1]... the notation is obscure...